packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -376,6 +377,7 @@ end
             kExtraFrontEndOptions: buildInfo.extraFrontEndOptions.join(','),
           kIosArchs: <DarwinArch>[DarwinArch.armv7, DarwinArch.arm64]
             .map(getNameForDarwinArch).join(' '),
+          kSdkRoot: await globals.xcode.sdkLocation(SdkType.iPhone),
```

Comment:
This is actually wrong and isn't building anything for the simulator.  Will fix in the next PR.